crates/ruff_python_semantic/src/binding.rs

```diff
@@ -135,6 +135,21 @@ impl<'a> Binding<'a> {
         self.flags.contains(BindingFlags::IN_EXCEPT_HANDLER)
     }
 
+    /// Return `true` if this [`Binding`] represents an explicit type alias
```

Comment:
I liked how you linked to the relevant pep in `checker/ast/mod.rs`. Could we do the same here?